core/templates/pages/exploration-editor-page/editor-tab/training-panel/training-data.service.spec.ts

```diff
@@ -20,15 +20,11 @@
 // training-data.service.ts is upgraded to Angular 8.
 import { AngularNameService } from
   'pages/exploration-editor-page/services/angular-name.service';
-import { AnswerClassificationResultObjectFactory } from
-  'domain/classifier/AnswerClassificationResultObjectFactory';
-import { AnswerGroupsCacheService } from 'pages/exploration-editor-page/editor-tab/services/answer-groups-cache.service';
+import { AnswerGroupsCacheService } from
+  /* eslint-disable-next-line max-len */
```

Comment:
I think we don't need pragma for imports with 80+ chars, @srijanreddy98 Can you please confirm?

Reply:
Yup, imports can violate the max-len rule